src/android/app/src/main/java/io/github/lime3ds/android/fragments/SetupFragment.kt

```diff
@@ -258,6 +266,30 @@ class SetupFragment : Fragment() {
                     R.string.add_games_warning_help
                 )
             )
+            add(
+                SetupPage(
+                    R.drawable.ic_palette,
+                    R.string.set_up_theme_settings,
+                    R.string.setup_theme_settings_description,
+                    0,
+                    true,
+                    R.string.setup_set_theme,
+                    {
+                        ThemeUtil.isDuringSetup = true
+                        showStaticThemeSelectionDialog()
+                    },
+                    false,
+                    false,
+                    {
+                        val preferences = PreferenceManager.getDefaultSharedPreferences(requireContext())
+                        if (preferences.getBoolean("ThemeSetupCompleted", false)) {
+                            StepState.STEP_COMPLETE
+                        } else {
+                            StepState.STEP_INCOMPLETE
+                        }
+                    }
+                )
+            )
```

Comment:
I would remove the extra setup page here. People have already mentioned about having many pages when setting up the emulator after a reinstallation, and this doesn't seem particularly necessary.

Reply:
I added it thinking most users won't be aware  of the newly added themw color option and because i see it on other apps setup pages as well 

If you still think it's unnecessary, than i can remove i guess

Reply:
A mention in the changelog is enough of a nudge imo.